src/components/FormattedTextDisplayArea.tsx

```diff
@@ -1,23 +1,77 @@
+import type { Directory } from "../types/directory";
 import { CodeMirrorEditor } from "./CodeMirrorEditor";
 import { CopyButton } from "./CopyButton";
 import { useFormat } from "./hooks/useFormat";
 
 export const FormattedTextDisplayArea = () => {
-	const { text, setText, formattedText } = useFormat();
+	const { text, setText, formattedText, formattedDirData } = useFormat();
 
+	
+	const createDirectoryElement = (dirData: Directory, dirIndex: number) => {
+		
+		const resultPrefixs = []
```

Comment:
@ShuntaToda 
スペルミスの警告が出ているので修正をお願いします！
```suggestion
		const resultPrefixes = []
```